assets/src/containers/IndexPage.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import SelectCard from '../components/SelectCard';
+import { Link, withRouter } from 'react-router-dom'
+import routes from '../routes/routes'
+
+const styles = theme => ({
+    root: {
+        flexGrow: 1,
+      },
+    wrapper: {
+        marginTop: theme.spacing.unit * 2,
+        flexDirection: "row",
+        justifyContent: "center"
+    },
+})
+
+function IndexPage (props) {
+    const { classes, location } = props;
```

Comment:
Could you change this to:
```javascript
const { classes, match } = props
const currentCourseId = match.params.courseId
```
The `match` property explicitly sends down the `courseId` param from `App.js`, making the implementation more robust.

Reply:
sure, done